### PR TITLE
Webpack 5 Config

### PR DESCRIPTION
**Pull request by jakemhiller**

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/webpack-config@4.0.2-alpha.05746b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5e8de7e209e50a6166e52a45--codecademy-styleguide.netlify.com>https://5e8de7e209e50a6166e52a45--codecademy-styleguide.netlify.com</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5e8de7e209e50a6166e52a45>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@17.7.3-alpha.745a1b.0<br/>@codecademy/webpack-config@5.0.7-alpha.745a1b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f8da565af028e6a65993160--gamut-preview.netlify.app>https://5f8da565af028e6a65993160--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f8da565af028e6a65993160>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@17.7.3-alpha.67ea27.0<br/>@codecademy/webpack-config@5.0.7-alpha.67ea27.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@17.7.4-alpha.09be1e.0<br/>@codecademy/webpack-config@5.0.7-alpha.09be1e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f8ddba24e54841131e45a93--gamut-preview.netlify.app>https://5f8ddba24e54841131e45a93--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f8ddba24e54841131e45a93>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@17.7.4-alpha.96a383.0<br/>@codecademy/webpack-config@5.0.7-alpha.96a383.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f8de62b8f43c81e0b7c47bb--gamut-preview.netlify.app>https://5f8de62b8f43c81e0b7c47bb--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f8de62b8f43c81e0b7c47bb>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@17.7.4-alpha.d46b20.0<br/>@codecademy/webpack-config@5.0.7-alpha.d46b20.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f8f39e3f0a31d31cf909878--gamut-preview.netlify.app>https://5f8f39e3f0a31d31cf909878--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f8f39e3f0a31d31cf909878>Deploy Logs</a></p>

File: packages/webpack-config/package.json [hunk 1]
```diff
@@ -23,30 +19,30 @@
...
     "cssnano": "^4.1.10",
-    "file-loader": "^3.0.1",
-    "mini-css-extract-plugin": "^0.12.0",
-    "node-sass": "^4.13.0",
-    "postcss-flexbugs-fixes": "^4.1.0",
-    "postcss-loader": "^3.0.0",
-    "sass-loader": "^7.1.0",
-    "style-loader": "^1.2.0",
-    "terser-webpack-plugin": "^4.1.0",
+    "file-loader": "^6.1.1",
```
Comment (jakemhiller) [inline packages/webpack-config/package.json:38]: TODO: remove this, it can be done in [webpack natively now](https://webpack.js.org/guides/asset-modules/)

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@17.7.4-alpha.3e6c77.0<br/>@codecademy/webpack-config@5.0.7-alpha.3e6c77.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f9891432f612721bf324c22--gamut-preview.netlify.app>https://5f9891432f612721bf324c22--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f9891432f612721bf324c22>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@17.7.4-alpha.08534d.0<br/>@codecademy/webpack-config@5.0.7-alpha.08534d.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f9892afb8755e19ce5b8660--gamut-preview.netlify.app>https://5f9892afb8755e19ce5b8660--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f9892afb8755e19ce5b8660>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@17.7.4-alpha.f25bf0.0<br/>@codecademy/webpack-config@5.0.7-alpha.f25bf0.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f98991234f3b722d3ffdfe9--gamut-preview.netlify.app>https://5f98991234f3b722d3ffdfe9--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f98991234f3b722d3ffdfe9>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f99b6eeb348911e7d19a13d--gamut-preview.netlify.app>https://5f99b6eeb348911e7d19a13d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f99b6eeb348911e7d19a13d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.9.3-alpha.077827.0<br/>@codecademy/gamut-labs@4.6.2-alpha.077827.0<br/>@codecademy/gamut@16.3.2-alpha.077827.0<br/>@codecademy/markdown-overrides@0.3.6-alpha.077827.0<br/>@codecademy/styleguide@17.10.3-alpha.077827.0<br/>@codecademy/tracking@0.1.2-alpha.077827.0<br/>@codecademy/webpack-config@5.1.1-alpha.077827.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.9.3-alpha.b6f65f.0<br/>@codecademy/gamut-labs@4.6.2-alpha.b6f65f.0<br/>@codecademy/gamut@16.3.2-alpha.b6f65f.0<br/>@codecademy/markdown-overrides@0.3.6-alpha.b6f65f.0<br/>@codecademy/styleguide@17.10.3-alpha.b6f65f.0<br/>@codecademy/tracking@0.1.2-alpha.b6f65f.0<br/>@codecademy/webpack-config@5.1.1-alpha.b6f65f.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.10.1-alpha.bc3b1a.0<br/>@codecademy/gamut-labs@4.6.3-alpha.bc3b1a.0<br/>@codecademy/gamut@16.3.3-alpha.bc3b1a.0<br/>@codecademy/markdown-overrides@0.3.6-alpha.bc3b1a.0<br/>@codecademy/styleguide@17.10.4-alpha.bc3b1a.0<br/>@codecademy/tracking@0.1.2-alpha.bc3b1a.0<br/>@codecademy/webpack-config@5.1.1-alpha.bc3b1a.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.10.1-alpha.9db381.0<br/>@codecademy/gamut-labs@4.6.3-alpha.9db381.0<br/>@codecademy/gamut@16.3.3-alpha.9db381.0<br/>@codecademy/markdown-overrides@0.3.6-alpha.9db381.0<br/>@codecademy/styleguide@17.10.4-alpha.9db381.0<br/>@codecademy/tracking@0.1.2-alpha.9db381.0<br/>@codecademy/webpack-config@5.1.1-alpha.9db381.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.10.1-alpha.c9eb05.0<br/>@codecademy/gamut-labs@4.6.3-alpha.c9eb05.0<br/>@codecademy/gamut@16.3.3-alpha.c9eb05.0<br/>@codecademy/markdown-overrides@0.3.6-alpha.c9eb05.0<br/>@codecademy/styleguide@17.10.4-alpha.c9eb05.0<br/>@codecademy/tracking@0.1.2-alpha.c9eb05.0<br/>@codecademy/webpack-config@5.1.1-alpha.c9eb05.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.10.2-alpha.0a1e45.0<br/>@codecademy/gamut-labs@5.2.2-alpha.0a1e45.0<br/>@codecademy/gamut@17.3.1-alpha.0a1e45.0<br/>@codecademy/markdown-overrides@0.3.9-alpha.0a1e45.0<br/>@codecademy/styleguide@18.5.1-alpha.0a1e45.0<br/>@codecademy/tracking@0.1.2-alpha.0a1e45.0<br/>@codecademy/webpack-config@5.2.1-alpha.0a1e45.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fad68da0a2d435723160a16--gamut-preview.netlify.app>https://5fad68da0a2d435723160a16--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fad68da0a2d435723160a16>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.10.2-alpha.0667b9.0<br/>@codecademy/gamut-labs@5.2.2-alpha.0667b9.0<br/>@codecademy/gamut@17.3.1-alpha.0667b9.0<br/>@codecademy/markdown-overrides@0.3.9-alpha.0667b9.0<br/>@codecademy/styleguide@18.5.1-alpha.0667b9.0<br/>@codecademy/tracking@0.1.2-alpha.0667b9.0<br/>@codecademy/webpack-config@5.2.1-alpha.0667b9.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fada4bf9e203a97fae9a243--gamut-preview.netlify.app>https://5fada4bf9e203a97fae9a243--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fada4bf9e203a97fae9a243>Deploy Logs</a></p>

File: packages/styleguide/package.json [hunk 2]
```diff
@@ -50,8 +50,8 @@
     "react": "16.13.1",
     "react-dom": "16.13.1",
     "typescript": "*",
-    "webpack": "4.44.2",
-    "webpack-cli": "4.0.0",
+    "webpack": "5.1.3",
+    "webpack-cli": "4.1.0",
```
Comment (jakemhiller) [inline packages/styleguide/package.json:54]: likely can remove this upgrade

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.12.1-alpha.e7b449.0<br/>@codecademy/gamut-illustrations@0.3.5-alpha.e7b449.0<br/>@codecademy/gamut-labs@7.5.6-alpha.e7b449.0<br/>@codecademy/gamut-styles@7.6.3-alpha.e7b449.0<br/>@codecademy/gamut-system@0.2.2-alpha.e7b449.0<br/>@codecademy/gamut@19.7.2-alpha.e7b449.0<br/>@codecademy/markdown-overrides@0.4.3-alpha.e7b449.0<br/>@codecademy/styleguide@21.6.3-alpha.e7b449.0<br/>@codecademy/tracking@0.2.2-alpha.e7b449.0<br/>@codecademy/webpack-config@5.2.2-alpha.e7b449.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.13.1-alpha.89864c.0<br/>@codecademy/gamut-illustrations@0.4.8-alpha.89864c.0<br/>@codecademy/gamut-labs@7.13.2-alpha.89864c.0<br/>@codecademy/gamut-styles@7.6.4-alpha.89864c.0<br/>@codecademy/gamut-system@0.2.2-alpha.89864c.0<br/>@codecademy/gamut@19.11.4-alpha.89864c.0<br/>@codecademy/markdown-overrides@0.4.4-alpha.89864c.0<br/>@codecademy/styleguide@21.19.1-alpha.89864c.0<br/>@codecademy/tracking@0.4.1-alpha.89864c.0<br/>@codecademy/webpack-config@5.2.2-alpha.89864c.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.13.1-alpha.662e79.0<br/>@codecademy/gamut-illustrations@0.4.8-alpha.662e79.0<br/>@codecademy/gamut-labs@7.13.2-alpha.662e79.0<br/>@codecademy/gamut-styles@7.6.4-alpha.662e79.0<br/>@codecademy/gamut-system@0.2.2-alpha.662e79.0<br/>@codecademy/gamut@19.11.4-alpha.662e79.0<br/>@codecademy/markdown-overrides@0.4.4-alpha.662e79.0<br/>@codecademy/styleguide@21.19.1-alpha.662e79.0<br/>@codecademy/tracking@0.4.1-alpha.662e79.0<br/>@codecademy/webpack-config@5.2.2-alpha.662e79.0<br/></pre>

**Pull request closed without merging**